common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisPilotingManager.java

```diff
@@ -769,16 +771,16 @@ public GlobalConsoleBlockEntity getCurrentConsole() {
     public void setCurrentConsole(GlobalConsoleBlockEntity newConsole) {
 
         if (this.currentConsole != null) {
-            this.currentConsole.getLevel().setBlockAndUpdate(this.currentConsole.getBlockPos(), this.currentConsole.getBlockState().setValue(GlobalConsoleBlock.POWERED, false));
+            if (this.currentConsole.getLevel().getBlockState(this.currentConsole.getBlockPos()).getBlock() instanceof GlobalConsoleBlock) {
```

Comment:
Code remanent, but make local variables for these, seems like getLevel is used a few times

Reply:
Not a remanent. Updated.